internal/cli/command/controlplane/up.go

```diff
@@ -348,6 +326,65 @@ func imageFileExists(options *cpContext, source string) (bool, error) {
 	}
 }
 
+func stringifyMap(m interface{}) interface{} {
```

Comment:
Maybe a json.marshal/unmarshal two-liner could replace this?

Reply:
json.marshal can't marshal `map[interface{}]interface{}`es, which are valid in yaml only (which is our data source)